R/export_cdisc.R

```diff
@@ -48,36 +57,34 @@ export_cdisc <- function(res_nca) {
   )
 
   # define columns needed for adpp
-  adpp_col <- c("STUDYID",
-                "DOMAIN",
-                "USUBJID",
-                "PPSEQ",
-                "PPCAT",
-                "PPGRPID",
-                "PPSPID",
-                "PPTESTCD",
-                "PPTEST",
-                "PPSCAT",
-                "PPREASND",
-                "PPSPEC",
-                "PPDTC",
-                "PPSTINT",
-                "PPENINT",
-                "SUBJID",
-                "SITEID",
-                "SEX",
-                "RACE",
-                "ACTARM",
-                "AAGE",
-                "AAGEU",
-                "TRT01P",
-                "TRT01A",
-                "AVAL",
-                "AVALC",
-                "AVALU")
+  adpp_cols <- c("STUDYID",
+                 "USUBJID",
+                 "PPSEQ",
+                 "PPGRPID",
+                 "PPSPID",
+                 "PARAMCD",
+                 "PARAM",
+                 "PARAMCAT",
+                 "PPSCAT",
+                 "PPREASND",
+                 "PPSPEC",
+                 "PPDTC",
+                 "PPSTINT",
+                 "PPENINT",
+                 "SUBJID",
+                 "SITEID",
+                 "SEX",
+                 "RACE",
+                 "ACTARM",
+                 "AAGE",
+                 "AAGEU",
+                 "TRT01P",
+                 "TRT01A",
+                 "AVAL",
+                 "AVALC",
+                 "AVALU")
```

Comment:
```suggestion
  adpp_cols <- c("STUDYID",
                 "USUBJID",
                 "PPSEQ",
                 "PPGRPID",
                 "PPSPID",
                 "PARAMCD",
                 "PARAM",
                 "PPCAT",
                 "PPSCAT",
                 "PPREASND",
                 "PPSPEC",
                 "PPDTC",
                 "PPSTINT",
                 "PPENINT",
                 "SUBJID",
                 "SITEID",
                 "SEX",
                 "RACE",
                 "ACTARM",
                 "AAGE",
                 "AAGEU",
                 "TRT01P",
                 "TRT01A",
                 "AVAL",
                 "AVALC",
                 "AVALU")
```

PPCAT needed, 
I removed PARAMCAT 

Somehow when I download the ADPP dataset, I get additional variables that we should remove: 
DRUG ANALYTE std_route PCSPEC

Could you please have a look?

Reply:
Those are the grouping variables for the intervals/parameter-calculations, I personally think it would be better to keep them for general use because internally in the App they make sense for identification

Reply:
For sure, we need PPCAT = PARAMCAT (PARAMCAT is not CDISC standard)

@Gero1999 If you look, we already have them:
We already have ANALYTE, which is PPCAT
We already have PCSPEC, which is PPSPEC (We should not have PC variables in PP or ADPP)
As mentioned in https://github.com/pharmaverse/aNCA/pull/306#discussion_r2026351421, DRUG is not useful for the PP and ADPP
std_route is not CDISC standard and anyways, it is not useful to have DRUG information in PP and ADPP

**That's why I would remove PARAMCAT, DRUG, ANALYTE, std_route, PCSPEC and add PPCAT**

Reply:
@Gero1999